book/src/passive-stake-delegation-and-rewards.md

```diff
@@ -0,0 +1,149 @@
+# Stake Delegation and Reward
+
+This design proposal focuses on the software architecture for the on-chain
+voting and staking programs.  Incentives for staking is covered in [staking
+rewardsd](staking-rewards.md).
+
+The current architecture requires an active vote for each delegated stake from
+the validator, and therefore does not scale to allow for replicator clients
+automatically delegate their rewards.
+ 
+The design proposes a new set of programs for voting and stake delegation, The
+proposed programs allow many stakes to passively earn rewards with a single
+validator vote without permission from the validator.
+
+## Current Design Problems
+
+In the current design each staker creates their own VoteState, and assigns a
+**delegate** in the VoteState that can submit votes.  Since the validator has to
+actively vote for each stake delegated to it, it can censor stakes by not voting
```

Comment:
What's its incentive to do that?

Reply:
To keep more rewards for it’s own cartel.